keras_core/ops/nn.py

```diff
@@ -959,6 +959,39 @@ def compute_output_spec(self, x):
 
 @keras_core_export(["keras_core.ops.one_hot", "keras_core.ops.nn.one_hot"])
 def one_hot(x, num_classes, axis=-1, dtype=None):
+    """
+    Converts integer tensor `x` into a one-hot tensor.
+
+    The one-hot encoding is a representation where each integer value is
+    converted into a binary vector with a length equal to `num_classes`,
+    and the index corresponding to the integer value is marked as 1, while
+    all other indices are marked as 0.
+
+    Args:
+        x (KerasTensor): Integer tensor to be encoded. The shape can be
```

Comment:
Rather than specifying the type before `:`, specify it after, like `x: Integer.`

Note that this input is NOT a KerasTensor. "Integer tensor" is the right description.